needle/modules/storage/data/files_mdm.py

```diff
@@ -0,0 +1,128 @@
+from core.framework.module import BaseModule
+from core.device.device import Device
+from core.utils.menu import choose_from_list_data_protection, choose_from_list, choose_yes_no
+from core.utils.utils import Utils
+from core.utils.printer import Colors
+
+class Module(BaseModule):
+    meta = {
+        'name': 'MDM dump',
+        'author': 'Oliver Simonnet (@mwr)',
+        'description': 'Locate and dump MDM configuration files',
+        'options': (
+            ('bulk', False, False, 'Dump all configuration files in a selected directory.'),
+            ('autosave', False, False, 'Automatically save files. (No save prompt)'),
+            ('output', True, True, 'File output directory.')
+        ),
+    }
+
+    # ==================================================================================================================
+    # UTILS
+    # ==================================================================================================================
+    def __init__(self, params):
+        BaseModule.__init__(self, params)
+        # Setting default output file
+        self.options['output'] = self.local_op.build_output_path_for_file(self, "mdm")
+
+    # Prompt user to select a directory, and return directory string
+    def select_dir(self, directories):
+        option = choose_from_list(directories)
+        if option is 'All': dirs_str =  ' '.join(directories[:-1]) 
+        else: dirs_str = option
+
+        self.printer.info("Selecting files from: [%s%s%s]" % (Colors.B, dirs_str.replace(" ", "\n\t"), Colors.N))
```

Comment:
This was implemented to ensure the output from line 33 is clean in the event they chose the "All" option (text wrapping is not elegant)
Instead of printing <dir-1> <dir-2> ... <dir-n>
prints:
    <dir-1>
    <dir-1>
    ....